mex/model/entities/bibliographic-resource.json

```
@@ -0,0 +1,510 @@
+            "sameAs": [
+                "http://purl.org/dc/terms/created"
+            ]
+        },
+        "journal": {
+            "default": [],
+            "description": "The periodical in which the article was published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "type": "array"
+        },
+        "keyword": {
+            "default": [],
+            "description": "A keyword or tag describing the resource ([DCAT, 2020-02-04](https://www.w3.org/TR/2020/REC-vocab-dcat-2-20200204/)).",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "sameAs": [
+                "http://www.w3.org/ns/dcat#keyword"
+            ],
+            "type": "array"
+        },
+        "language": {
+            "default": [],
+            "description": "The language in which the publication was written.",
+            "items": {
+                "$ref": "/schema/entities/concept#/identifier",
+                "examples": [
+                    "https://mex.rki.de/item/language-1"
+                ],
+                "useScheme": "https://mex.rki.de/item/language"
+            },
+            "sameAs": [
+                "http://purl.org/dc/terms/language"
+            ],
+            "type": "array"
+        },
+        "license": {
+            "anyOf": [
+                {
+                    "$ref": "/schema/entities/concept#/identifier",
+                    "examples": [
+                        "https://mex.rki.de/item/license-1"
+                    ],
+                    "useScheme": "https://mex.rki.de/item/license"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "A legal document giving official permission to do something with the publication ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "sameAs": [
+                "http://purl.org/dc/terms/license"
+            ]
+        },
+        "pages": {
+            "anyOf": [
+                {
+                    "examples": [
+                        "1",
+                        "45-67",
+                        "45 - 67",
+                        "II",
+                        "XI",
+                        "10i"
+                    ],
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The range of pages or a single page."
+        },
+        "publicationYear": {
+            "anyOf": [
+                {
+                    "$comment": "year",
+                    "examples": [
+                        "2011",
+                        "1982",
+                        "1810"
```

```suggestion
                        "2024"
```

so it is inline with the other "year" field examples